.travis/test_cython_files.sh

```diff
@@ -6,10 +6,20 @@ find jsonobject -iname '*.so' -delete
 python setup.py build_ext --inplace
 
 git update-index -q --refresh
-if git diff-index --quiet HEAD --; then
```

Comment:
`diff-index` should be the same as `diff`, since both `HEAD` and `--` are specified.  Changing just to show that this check uses the same cmd as the output below